tests/pingpong-shared/enc/enc.cpp

```diff
@@ -6,7 +6,7 @@
 
 void Ping(const char* in, char* out)
 {
-    Pong(in, out);
+    Pong(in, out, sizeof(out));
```

Comment:
BUG: this line is incorrect since sizeof(out) is sizeof(char*) not size of the buffer.  Instead, have the caller pass in the length as a third argument to Ping()